tests/test_server_endpoints.py

```
@@ -77,6 +77,174 @@ def test_api_chat_all_fields(
+def test_api_chat_with_images(
+    mock_get_global_instructions,
+    mock_create_toolcalling_llm,
+    client,
+):
+    """Test /api/chat endpoint with image analysis support."""
+    mock_ai = MagicMock()
+
+    # Capture the messages passed to the LLM
+    captured_messages = []
+
+    def capture_messages(messages, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `kwargs`

(ARG001)